webui/react/src/pages/TrialDetails/TrialDetailsOverview.tsx

```diff
@@ -79,8 +80,7 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
       <TrialDetailsWorkloads
         defaultMetrics={defaultMetrics}
         experiment={experiment}
-        metricNames={metricNames}
-        metrics={metrics}
+        selectedMetrics={metrics}
         settings={settings}
```

Comment:
there should be `allMetrics` here or similar (and it should be used in the component)